### ISSUE TITLE
[BUG] documentation for harbor.yaml has wrong subkey for internal_tls

### ISSUE THREAD

**Issue by GeoffMillerAZ** [goharbor/website#99]
https://github.com/goharbor/website/blob/master/docs/install-config/configure-yml-file.md

On that page it shows:

`internal_tls` with `subkey` certificate for `The path to the directory that contains internal certs and keys`. The subkey is not `certificate`

resulting error:
```bash
[Step 4]: preparing harbor configs ...
prepare base dir is set to /repay/harbor/installer/harbor
Traceback (most recent call last):
  File "main.py", line 15, in <module>
    cli()
  File "/usr/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/src/app/commands/prepare.py", line 38, in prepare
    config_dict = parse_yaml_config(conf, with_notary=with_notary, with_clair=with_clair, with_trivy=with_trivy, with_chartmuseum=with_chartmuseum)
  File "/usr/src/app/utils/configs.py", line 349, in parse_yaml_config
    internal_tls_config['dir'],
KeyError: 'dir'
```

resolution:
change the subkey from `certificate` to `dir`

**Comment by GeoffMillerAZ** [comment]
https://github.com/goharbor/website/pull/100

**Comment by a-mccarthy** [comment]
closed by #100
